tmeister/static/src/toggles/delete/delete-modal-metrics.component.js

```diff
@@ -1,5 +1,14 @@
 import React from 'react'
-import { sumBy } from 'lodash'
+import { sumBy, get } from 'lodash'
+import dayjs from 'dayjs'
+
+const calendarSettings = {
+  sameDay: '[Today]',
+  nextDay: '[Tomorrow]',
+  lastDay: '[Yesterday]',
+  lastWeek: '[Last] dddd',
+  sameElse: 'MM/DD/YYYY'
+}
```

Comment:
Override behavior for `calendar` api so it doesn't show the time

Reply:
not including `nextWeek`?